pdq/instability.go

```diff
@@ -0,0 +1,20 @@
+package pdq
+
+import (
+	"fmt"
+)
+
+func CalcInstability(outgoingDependenciesNum int, incomingDependenciesNum int) (float64, error) {
+	if outgoingDependenciesNum < 0 || incomingDependenciesNum < 0 {
+		return -1, fmt.Errorf("outgoingDependenciesNum %v, incomingDependenciesNum %v; values must be >= 0", outgoingDependenciesNum, incomingDependenciesNum)
```

Comment:
its better to use `%d` instead of `%v` for formatting integers in erro msg, as this more accurately reflects the data typ